compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -1128,10 +1128,6 @@ pub fn describe_flag_categories(early_dcx: &EarlyDiagCtxt, matches: &Matches) ->
         early_dcx.early_warn("the `-Cno-stack-check` flag is deprecated and does nothing");
     }
 
-    if cg_flags.iter().any(|x| x.starts_with("inline-threshold")) {
```

Comment:
Previously this will warn for any possible flags with `inline-threshold` prefix, but not now.